examples/arkode/C_serial/ark_kepler_plot.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
I think we should use `python3` for new scripts

Reply:
I would agree except that I believe `python` and `python3` are nonstandard and `python` may refer to python version 3 in most modern installations, and some installations (Windows in particular) only have `python` (referring to python version 3).